model/client4.go

```diff
@@ -386,6 +386,10 @@ func (c *Client4) GetComplianceReportRoute(reportId string) string {
 	return fmt.Sprintf("/compliance/reports/%v", reportId)
 }
 
+func (c *Client4) GetComplianceReportDownloadRoute(reportId string) string {
+	return fmt.Sprintf("/compliance/reports/%v/download", reportId)
+}
```

Comment:
I see you followed the pattern used in the above function, but let's update both `GetComplianceReportDownloadRoute` and `GetComplianceReportRoute` to use `GetComplianceReportsRoute`.